Controller/ContentManagement/TemplateController.php

```diff
@@ -2,25 +2,29 @@
 
 namespace EMS\CoreBundle\Controller\ContentManagement;
 
-use EMS\CoreBundle\Controller\AppController;
-use EMS\CoreBundle;
-use EMS\CoreBundle\Repository\ContentTypeRepository;
 use Doctrine\ORM\EntityManager;
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
-use Symfony\Bundle\FrameworkBundle\Controller\Controller;
-use Symfony\Component\HttpFoundation\Request;
-use EMS\CoreBundle\Form\Form\TemplateType;
+use Doctrine\ORM\OptimisticLockException;
+use Doctrine\ORM\ORMException;
+use EMS\CoreBundle\Controller\AppController;
 use EMS\CoreBundle\Entity\Template;
+use EMS\CoreBundle\Form\Form\TemplateType;
+use EMS\CoreBundle\Repository\ContentTypeRepository;
 use EMS\CoreBundle\Repository\TemplateRepository;
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 
 class TemplateController extends AppController
 {
     /**
-     * @Route("/template/{type}", name="template.index")
+     * @param string $type
```

Comment:
You should not document @param and @return when you can add them in the php code.

Reply:
You can desactivate this autocreation in your IDE (because your IDE does not keep this doc up to date and we use php7)

Reply:
see 6dbb28ee10671cd9cb5bcaa855da40f37286dafc